docs/orchestration/recipes/third_party_auth.md

```diff
@@ -0,0 +1,86 @@
+# Third Party Authentication
```

Comment:
This doc is really well written! If we keep it in this location do you think we should expand the other "recipes" in this section to become more like this? The others feel to me more like bits of code while this doc reads like a concept doc.

Reply:
Hmmm I don't think we need to change them immediately but I think the more guidance we can give in the other docs the better